cheri_compressed_cap.h

```diff
@@ -93,6 +98,25 @@ enum CC256_OTypes {
     CC256_SPECIAL_OTYPE(LAST_NONRESERVED_OTYPE, 15),
 };
 
+#define LS_SPECIAL_OTYPES_COMMON(ITEM, ...)                                                                            \
+    ITEM(OTYPE_UNSEALED, __VA_ARGS__)                                                                                  \
+    ITEM(OTYPE_SENTRY, __VA_ARGS__)
+
+#ifdef IS_MORELLO
+
+#define LS_SPECIAL_OTYPES(ITEM, ...)                                                                                   \
+    LS_SPECIAL_OTYPES_COMMON(ITEM, __VA_ARGS__)                                                                        \
+    ITEM(OTYPE_LOAD_PAIR_BRANCH, __VA_ARGS__)                                                                          \
```

Comment:
Maybe we should reserve those values for CHERI-RISC-V too?

Reply:
I see no harm, we have plenty of reserved types spare, and if we do ever adopt them for RISCV keeping the numbers the same is neater. I will change it if nobody objects. Also, only RISCV and not all CHERI?

Reply:
INDIRECT_SENTRY and INDIRECT_PAIR or similar would be the CHERI names for them though, not LB/LPB (even if expanded) that match some of the terminology in ISAv8. This is what reserved 2 and 3 are reserved for anyway, FWIW, this would just be giving names to them. And yes, just make it all CHERIs (even Morello, it can use the generic CHERI name in QEMU rather than their own invention).

Reply:
All of CHERI, but MIPS is eventually going away so it doesn't matter as much.